transport/channel.go

```diff
@@ -44,20 +44,27 @@ type Channel interface {
 	// MAY be called multiple times for a request. The returned outbound MUST
 	// have already been started.
 	GetUnaryOutbound() UnaryOutbound
+	GetOnewayOutbound() OnewayOutbound
```

Comment:
Mention the expected behavior if this channel doesn't provide a oneway/unary outbound.